src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -345,7 +345,7 @@ private class TaskLauncherActor(
       inProgress = tasksToLaunch > 0 || inFlightTaskOperations.nonEmpty,
       tasksLeftToLaunch = tasksToLaunch,
       finalTaskCount = tasksToLaunch + taskLaunchesInFlight + tasksLaunched,
-      tasksLost = tasksMap.values.count(_.mesosStatus.fold(false)(_.getState == Mesos.TaskState.TASK_LOST)),
+      tasksLost = tasksMap.values.count(value => value.mightBeLost),
```

Comment:
In ```QueuedTaskInfo``` do we really need only the number of unreachable tasks for the val ```tasksLost```? Gone and Dropped are unimportant?